### PR TITLE
Handle forge update.

### PR DESCRIPTION
**Pull request by ThatGravyBoat**

### CODE REVIEW

**Review by MsRandom — APPROVED** [review]

File: forge/src/main/java/earth/terrarium/techarium/forge/extensions/PlatformHelperImpl.java [hunk 1]
```diff
@@ -23,7 +23,7 @@ public static boolean isDevelopmentEnvironment() {
 
 	@ExtensionImplementation
 	public static void openMenu(ServerPlayer player, ExtraDataMenuProvider provider) {
-		NetworkHooks.openGui(player, provider, (data) -> provider.writeExtraData(player, data));
+		NetworkHooks.openScreen(player, provider, (data) -> provider.writeExtraData(player, data));
```
Comment (CodexAdrian) [inline forge/src/main/java/earth/terrarium/techarium/forge/extensions/PlatformHelperImpl.java:26]: Seems a lil dubious

File: forge/src/main/java/earth/terrarium/techarium/forge/extensions/ForgeRegistryHelper.java [hunk 2]
```diff
@@ -37,7 +37,7 @@ public static <E extends AbstractContainerMenu> MenuType<E> createMenuType(Regis
     }
 
     @ExtensionImplementation
-    public static ResourceKey<? extends Registry<MultiblockStructure>> getMultiblockRegistryKey() {
+    public static ResourceKey<Registry<MultiblockStructure>> getMultiblockRegistryKey() {
```
Comment (CodexAdrian) [inline forge/src/main/java/earth/terrarium/techarium/forge/extensions/ForgeRegistryHelper.java:40]: I like question marks

File: forge/src/main/java/earth/terrarium/techarium/forge/datagen/DatagenEvenHandler.java [hunk 3]
```diff
@@ -16,9 +16,9 @@
 import net.minecraft.resources.ResourceLocation;
 import net.minecraftforge.common.data.ExistingFileHelper;
 import net.minecraftforge.common.data.JsonCodecProvider;
+import net.minecraftforge.data.event.GatherDataEvent;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.common.Mod;
-import net.minecraftforge.forge.event.lifecycle.GatherDataEvent;
```
Comment (CodexAdrian) [inline forge/src/main/java/earth/terrarium/techarium/forge/datagen/DatagenEvenHandler.java:21]: idk man, i liked where it was before, put it back

**Review by CodexAdrian — APPROVED** [review]

File: forge/src/main/java/earth/terrarium/techarium/forge/extensions/PlatformHelperImpl.java (hunk 1 above)
Comment (ThatGravyBoat) [reply forge/src/main/java/earth/terrarium/techarium/forge/extensions/PlatformHelperImpl.java:26]: ur mom

**Pull request merged**